src/main/java/com/devcourse/checkmoi/domain/user/repository/CustomUserRepositoryImpl.java

```diff
@@ -35,6 +35,11 @@ public UserInfoWithStudy findUserInfoWithStudy(Long userId) {
             .build();
     }
 
+    @Override
+    public int userJoinedStudies(Long userId) {
```

Comment:
userJoinedStuies 라고 하면 반환타입이 study일 것 같은 네이밍이라고 느껴지는데 혹시
```suggestion
    public int getJoinedStudySize(Long userId) {
```
와 같은 네이밍은 어떠신가요?